app/src/main/java/com/example/erabook/data/models/AccessInfo.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.erabook.data.models
+
+import Epub
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class AccessInfo(
+    var country: String? = null,
```

Comment:
Because when I take the json response and use a tool to convert it to all the kotlin data classes it makes them with var, I forgot to double check I'll rename all of them now.